app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -71,24 +68,14 @@ fun MessageComposer(
             onMessageComposeInputStateChanged = onMessageComposerInputStateChange
         )
 
-        LaunchedEffect(messageText) {
-            messageComposerState.messageText = messageComposerState.messageText.copy(messageText)
-        }
-
         MessageComposer(
             content = content,
             keyboardHeight = keyboardHeight,
             messageComposerState = messageComposerState,
-            messageText = messageComposerState.messageText,
-            onMessageChanged = {
-                // we are setting it immediately in the UI first
-                messageComposerState.messageText = it
-                // we are hoisting the TextFieldValue text value up to the parent
-                if (messageText != it.text) {
-                    onMessageChanged(it.text)
-                }
+            onSendButtonClicked = {
```

Comment:
Maybe wrapping lambdas in `remember` will also decrease unnecessary recompositions?

Reply:
cant hurt :P